Objects/Converters/ConverterRevit/ConverterRevitTests/ConverterRevitTestsShared/SpeckleConversionTest.cs

```diff
@@ -194,6 +195,7 @@ await APIContext
           {
             converter.Report.LogConversionError(new Exception(e.Message, e));
           }
+#pragma warning restore CA1031 // Do not catch general exception types
```

Comment:
As with other statements, if this is not top level handler, then !IsFatal() and log